lib/components/term.tsx

```diff
@@ -211,6 +212,7 @@ export default class Term extends React.PureComponent<
         )
       );
       this.term.open(this.termRef);
+      this.term.loadAddon(new ImageAddon('./node_modules/xterm-addon-image/lib/xterm-addon-image-worker.js'));
```

Comment:
The worker abstraction is gone for now, as it created to many frictions on integrator side. So if you are fine with the addon's default options, thats only `new ImageAddon()` currently.

Reply:
Do those newer versions of xterm-addon-image work with xterm v4 as well? I tried, but didn't get it to work. That is what Hyper is based on at the moment.

Reply:
Ah nope, the older 4.x versions need older addon versions as well, still including the worker.

Reply:
@jerch thanks for reviewing this. I'm trying to upgrade to xterm v5, though there are some issues with ligatures on v5, for which I've opened few prs on xterm repo. I think we can hold off on image support till we can get to v5.

Reply:
Understandable. Will follow the topic and keep this PR up-to-date with the canary branch and its xterm version. 

Do you plan on including xterm v5 directly in the v4.0.0 release or at a later time? Any estimated release date?

Reply:
In v4 itself, no fixed release date at this point, but working on few significant things and plan to release soon after.

Reply:
I'm waiting for https://github.com/xtermjs/xterm.js/pull/4386 and https://github.com/xtermjs/xterm.js/pull/4387 to be closed, then we can upgrade to xterm v5